### PR TITLE
Added LaLaLa feature

### PR DESCRIPTION
**Pull request by devendrarajput**
Started with La feature, but noticed that you created LaLa feature so decided to take advantage of it and make an awesome LaLaLa feature

### CODE REVIEW

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+La
+took benefit of LaLa feature to make LaLaLa
```
Comment (yashrajput) [inline file2.txt:3]: This looks really cool. Can you please put some comment around your code

Comment (devendrarajput) [reply file2.txt:3]: Been there done that

**Review by yashrajput — APPROVED** [review]
Good job father

**Comment by yashrajput** [comment]
### Short Summary

_**Hello! I am the King of the Lions! Also known as The Blue Knight! How are you? This code is awesome and you should try it! Thx!**_

**Pull request merged**